### PR TITLE
Don't branch while releasing

### PR DESCRIPTION
**Pull request by raboof**
Don't introduce a new branch while releasing, so the release is just a tag on
master.

See also #26675 .

### CODE REVIEW

File: project/scripts/release [hunk 1]
```diff
@@ -294,9 +267,9 @@ fi
 # build the release
 echolog "Building the release..."
 if [ ! $dry_run ]; then
-  RELEASE_OPT="-Dakka.genjavadoc.enabled=true -Dpublish.maven.central=true"
+  RELEASE_OPT="-Dakka.build.version=${version} -Dakka.genjavadoc.enabled=true -Dpublish.maven.central=true"
 else
-  RELEASE_OPT="-Dakka.genjavadoc.enabled=true"
+  RELEASE_OPT="-Dakka.build.version=${version} -Dakka.genjavadoc.enabled=true"
```
Comment (patriknw) [inline project/scripts/release]: nice, this removes one complexity from the release build

**Review by patriknw — APPROVED** [review]
LGTM

**Review by helena — APPROVED** [review]
LGTM but do we need to modify the contrib doc at all in the new releasing section? I don't think so.

**Comment by raboof** [comment]
> do we need to modify the contrib doc at all in the new releasing section? I don't think so

I looked over it and indeed it doesn't seem so

**Pull request merged**